src/components/SafeListSidebar/SafeList/index.tsx

```diff
@@ -45,70 +42,96 @@ const useStyles = makeStyles({
       paddingLeft: '44px',
     },
   },
-  noIcon: {
-    visibility: 'hidden',
-    width: '28px',
-  },
 })
 
 type Props = {
-  currentSafeAddress: string | undefined
-  safes: SafeRecordWithNames[]
-  ownedSafes: string[]
   onSafeClick: () => void
 }
 
-const isAddressAdded = (addedSafes: SafeRecordWithNames[], address: string): boolean => {
-  return addedSafes.some((safe) => sameAddress(safe.address, address))
+const isAddressAdded = (addedSafes: SafeRecordWithNames[], address: string): boolean =>
+  addedSafes.some((safe) => sameAddress(safe.address, address))
+
+const getSafesStoredLocally = (
+  networkId: ETHEREUM_NETWORK,
+  safes: SafeRecordWithNames[],
+  ownedSafes: string[],
+): SafeRecordWithNames[] => {
+  let otherSafes: SafeRecordWithNames[] = []
+
+  const name = getNetworkName(networkId)
+  const safeString = localStorage.getItem(getSafesKey(name))
```

Comment:
This should be using `loadFromStorage`. Please don't hack the localStorage directly.

Reply:
I have had to modify `loadFromStorage` to allow me to use it:

![image](https://user-images.githubusercontent.com/20442784/133748349-e67e2b7a-cb47-442a-bb51-1caa0ef2879e.png)

![image](https://user-images.githubusercontent.com/20442784/133748404-8c1b6ac8-7b89-4f34-9d36-ebb603ea2b30.png)

What are your thoughts?

Reply:
Sounds good, we'll probably need `loadFromStorage` to have this ability in other places.